app/frontend/components/common/footer.js

```diff
@@ -32,9 +32,10 @@ const Footer = connect(
           'div',
           {class: 'social'},
           h(
-            'a',
+            'button',
```

Comment:
Any particular reason why this is the only link that's actually a `<button>`? The `<a>` tag has also a permitted value for `role="button"` if you'd like to keep that.

Reply:
Discussed personally. Solved.